### PR TITLE
[WebView] Support opening WebView links in external browser on iOS

### PR DESCRIPTION
**Pull request by masonicboom**

### CODE REVIEW

File: React/Views/RCTWebView.h [hunk 1]
```diff
@@ -17,6 +17,7 @@
 @property (nonatomic, assign) UIEdgeInsets contentInset;
 @property (nonatomic, assign) BOOL shouldInjectAJAXHandler;
 @property (nonatomic, assign) BOOL automaticallyAdjustContentInsets;
+@property (nonatomic, assign) BOOL openLinksInExternalBrowser;
```
Comment (a2) [inline React/Views/RCTWebView.h]:
```objc
BOOL openLinksInExternalBrowser;
```

Comment (masonicboom) [inline React/Views/RCTWebView.h]: @a2 What does that mean? I don't know Objective-C; does that line have a mistake?

**Comment by arunshan** [comment]
I tried using your PR and I am not able to get the links to open in external browser. I think this only works with <a href=''/> but not with jquery on click listeners.

**Comment by masonicboom** [comment]
@arunshan can you post some code that exposes the issue you're seeing?

**Comment by facebook-github-bot-8** [comment]
Thank you for reporting this issue and appreciate your patience. We've  notified the core team for an update on this issue. We're looking for a response within the next 30 days or the issue may be closed.

**Comment by paramaggarwal** [comment]
@arunshan Yes, that is expected. The URL is never seen by the native webview itself if the click is handled completely by jQuery/JS. It would be an AJAX call and not a complete page reload.

All, any other concerns with this PR?

**Comment by satya164** [comment]
Any updates on this?

**Comment by satya164** [comment]
Closing since no activity on the PR. let's re-open if you wanna work on it again.

**Pull request closed without merging**